recipes-core/os-release/os-release.bbappend

```diff
@@ -17,7 +17,6 @@ OS_RELEASE_FIELDS += "OMNECT_OS_GIT_BRANCH"
 OS_RELEASE_FIELDS += "OMNECT_OS_GIT_REPO"
 OS_RELEASE_FIELDS += "OMNECT_OS_VERSION"
 
-OS_RELEASE_FIELDS += "OMNECT_TOOLS"
 OS_RELEASE_FIELDS += "OMNECT_DEVEL_TOOLS"
```

Comment:
imho should only be added in non-release image

Reply:
agreed